SpatialGDK/Source/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -64,6 +64,10 @@ void USpatialWorkerConnection::ConnectToReceptionist(bool bConnectAsClient)
 	FTCHARToUTF8 WorkerTypeCStr(*ReceptionistConfig.WorkerType);
 	ConnectionParams.worker_type = WorkerTypeCStr.Get();
 	ConnectionParams.enable_protocol_logging_at_startup = ReceptionistConfig.EnableProtocolLoggingAtStartup;
+	if (ConnectionParams.enable_protocol_logging_at_startup)
+	{
+		ConnectionParams.protocol_logging.log_prefix = WorkerTypeCStr.Get();
```

Comment:
Could we add a command line flag for the log prefix, and if it's not specified we default to the worker name?